child_compassion/mappings/child_assessment_mapping.py

```diff
@@ -0,0 +1,42 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2016 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Maxime Beck <mbcompte@gmail.com>
+#
+#    The licence is in the file __openerp__.py
+#
+##############################################################################
+from openerp.addons.message_center_compassion.mappings.base_mapping import \
+    OnrampMapping
+from datetime import datetime
+
+
+class ChildAssessmentMapping(OnrampMapping):
+    """
+    Child Assessment Mapping
+    """
+    ODOO_MODEL = 'compassion.child.cdpr'
+    MAPPING_NAME = 'beneficiary_cdpr'
+
+    CONNECT_MAPPING = {
+        'AssesmentType': 'assesment_type',
+        'CognitiveOutcomeScore': 'cognitive_score',
+        'CompletionDate': 'date',
+        'PhysicalOutcomeScore': 'physical_score',
+        'SocioEmotionalOutcomeScore': 'sociological_score',
+        'SpiritualOutcomeScore': 'spiritual_score',
+        'Age': 'age',
+        'Beneficiary_GlobalID': 'child_id',
```

Comment:
you can replace the mapping value with : `('child_id.global_id', 'compassion.child')`
which will allow you to avoid manually searching for the child in the `process_commkit` method.